hack/Makefile

```diff
@@ -51,4 +51,7 @@ generate-bundle:
 
 generate-from-swagger: lint-swagger generate-go-client generate-go-server validate-swagger-file --remove-dashes-and-dots
 
-generate: generate-from-swagger generate-events generate-mocks generate-configuration
+generate-vendor: 
+	go mod vendor && git diff --exit-code vendor
```

Comment:
no need for the `` && git diff --exit-code vendor``, locally it doesn't really do as intended
CI should handle comparing actual ``git diff``